beesdoo_shift/models/planning.py

```diff
@@ -112,7 +112,8 @@ def _generate_next_planning(self):
             )
             return
 
-        planning.task_template_ids._generate_task_day()
+        task_recset = planning.task_template_ids._generate_task_day()
+        task_recset = task_recset.create_all()
```

Comment:
I'm not entirely sure this is a good pattern. Doing one thing should ideally be done in one line.

Would it be possible to do something like this:

- `_generate_task_day()` exhibits the behaviour of the two lines created above.
- A new method is created that exhibits the behaviour of `_generate_task_day()` sans `create_all()`. This method is called internally by `_generate_task_day()`.

Before implementing that suggestion, verify that it's actually a good idea :) I don't know the context behind this change.